packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/ReflectUtils.kt

```diff
@@ -58,6 +58,7 @@ private fun <T : Any> T.getDeclaredMethodRecursively(
         val lookingInClass = classesToSearch.removeAt(0)
         classesSearched.add(lookingInClass)
         method = try {
+            var methods = lookingInClass.declaredMethods
```

Comment:
is not used it seems

Reply:
Ah, thanks. I was trying to figure out how to call something via reflection and forgot to remove this.